### PR TITLE
Feature/seed

### PR DESCRIPTION
**Pull request by jkt001**
- [x] Seed Movies
- [x] Seed Movie Crews
- [x] Seed Applications
- [x] Seed Application Crews

### CODE REVIEW

**Comment by jkt001** [comment]
fix conflict:

reason: corrupted file header was fixed in master, new data seeded in feature/seed. Fix: redump-data to multimedia_data and crew_data using correct file header and merge master to feature/seed

**Comment by jkt001** [comment]
yang tadi gw salah dump data nya... makanya migrate nya error (cc: @mikelimantara , @digawp ) gw td dump data multimedia+crew ke multimedia which explains why itu error... now ud fixed... maaf2... sorry

**Comment by 9gix** [comment]
@jkt001 tolong di tick kalo semua checklist nya uda selesai. Nanti kalo sudah confirm kasi label `ready` aja di PR ini.

Tolong di summarize. What has changed? 
What happened to the data? Why some data are removed and some are added?

**Comment by jkt001** [comment]
labeled. waiting for merge.

**Comment by jkt001** [comment]
summary:

-seed for applications
- waktu di seed, file header di branch ini blm di benerin...
- master di merge ke branch ini, buat benerin file header

ada revert itu karena:
1. tadi gw add movie sblm pull nathan punya, ud gw commit
2. gw revert commit itu
3. gw pake nathan punya, terus sadar bakal conflict kl merge ke master
4. gw merge master ke sini (bener conflict, masalah di file header (tanya diga: tadi conflict ga muncul apa2 di filenya padahal)
5. gw redump data ke file baru (iya ga pake powershell, tenang aja.. pake git bash)
6. migration OK, udah di coba

File: multimedia/migrations/0003_seed_multimedia.py [hunk 1]
```diff
@@ -27,6 +29,9 @@ def delete_multimedia(apps, schema_editor):
     Book.objects.all().delete()
     Music.objects.all().delete()
     Album.objects.all().delete()
+    Movie.objects.all().delete()
+    Application.objects.all().delete()
+    Multimedia.objects.all().delete()
```
Comment (9gix) [inline multimedia/migrations/0003_seed_multimedia.py:34]: @jkt001 Duplicate

File: multimedia/migrations/0002_create_model.py [hunk 2]
```diff
@@ -129,4 +129,26 @@ class Migration(migrations.Migration):
             },
             bases=('multimedia.multimedia',),
         ),
+        migrations.CreateModel(
+            name='Movie',
```
Comment (9gix) [inline multimedia/migrations/0002_create_model.py:133]: @jkt001 what's the bases?

File: multimedia/migrations/0002_create_model.py [hunk 3]
```diff
@@ -129,4 +129,26 @@ class Migration(migrations.Migration):
...
+                ('multimedia', models.OneToOneField(primary_key=True, to='multimedia.Multimedia', parent_link=True, serialize=False)),
+                ('duration', models.IntegerField(null=True)),
+            ],
+            options={
+                'db_table': 'movie',
+                'managed': False,
+            }
+        ),
+        migrations.CreateModel(
+            name='Application',
```
Comment (9gix) [inline multimedia/migrations/0002_create_model.py:144]: @jkt001 missing bases

**Pull request merged**